### PR TITLE
Make title bar on macOS transparent

### PR DESCRIPTION
**Pull request by 123Duo3**

### CODE REVIEW

**Review by Sanlorng — APPROVED** [review]

File: gallery/src/commonMain/kotlin/com/konyaco/fluent/gallery/App.kt [hunk 1]
```diff
@@ -36,10 +37,12 @@ import com.konyaco.fluent.gallery.screen.settings.SettingsScreen
 import com.konyaco.fluent.icons.Icons
 import com.konyaco.fluent.icons.regular.Settings
 import com.konyaco.fluent.surface.Card
+import androidx.compose.ui.window.*
 
 @Composable
-fun App() {
-    Row(Modifier.fillMaxSize()) {
+fun App(titleBarHeight: Dp) {
```
Comment (terrakok) [inline gallery/src/commonMain/kotlin/com/konyaco/fluent/gallery/App.kt:43]: Don't forget to update other targets as well 😉

Comment (Sanlorng) [reply gallery/src/commonMain/kotlin/com/konyaco/fluent/gallery/App.kt:43]: > Don't forget to update other targets as well 😉 

Windows support it on Dev branch

Comment (terrakok) [reply gallery/src/commonMain/kotlin/com/konyaco/fluent/gallery/App.kt:43]: I meant the app function is used for all targets such as android or web, isn't it?

Comment (Sanlorng) [reply gallery/src/commonMain/kotlin/com/konyaco/fluent/gallery/App.kt:43]: > I meant the app function is used for all targets such as android or web, isn't it?

Yes, i fixed this problem in `sync_master_target` branch.

**Pull request merged**